### PR TITLE
Hotfix/45 change dates to js object

### PR DESCRIPTION
**Pull request by RamonG92**

### CODE REVIEW

File: src/entities/run.entity.ts [hunk 1]
```diff
@@ -16,31 +16,27 @@ export class Run {
 
     @Column({
         name: 'time_o2_start',
-        type: 'timestamp',
-        default: () => 'CURRENT_TIMESTAMP',
+        precision: 0,
     })
-    timeO2Start: Date;
+    timeO2Start: Date = new Date();
```
Comment (misharigot) [inline src/entities/run.entity.ts]: Check all the date fields in this class as well if `new Date()` is appropriate.

File: src/entities/run_quality_history.entity.ts [hunk 2]
```diff
@@ -30,10 +30,9 @@ export class RunQualityHistory {
 
     @Column({
         name: 'change_time',
-        type: 'timestamp',
-        default: () => 'CURRENT_TIMESTAMP',
+        precision: 0,
     })
-    changeTime: Date;
+    changeTime: Date = new Date();
```
Comment (misharigot) [inline src/entities/run_quality_history.entity.ts]: Idem.

File: src/migration/1539014853555-CreateDatabase.ts [hunk 3]
```diff
@@ -0,0 +1,104 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
```
Comment (misharigot) [inline src/migration/1539014853555-CreateDatabase.ts]: All these migration files seem like a bit much for an initial migration, i.e. creating the db for the first time. Lets discuss if there's a better solution.

File: src/entities/epn_role_session.entity.ts [hunk 4]
```diff
@@ -38,15 +38,13 @@ export class EpnRoleSession {
 
     @Column({
         name: 'session_start',
-        type: 'timestamp',
-        default: () => 'CURRENT_TIMESTAMP',
+        precision: 0,
     })
-    sessionStart: string;
+    sessionStart: Date = new Date();
```
[the remark this answers is not in the document]
Comment (RamonG92) [reply src/entities/epn_role_session.entity.ts]: Changed so it doesn't give a default. Can be manually created if needed.

File: src/entities/epn_role_session.entity.ts [hunk 5]
```diff
@@ -38,15 +38,13 @@ export class EpnRoleSession {
...
 
     @Column({
         name: 'session_end',
-        type: 'timestamp',
-        default: () => 'CURRENT_TIMESTAMP',
+        precision: 0,
     })
-    sessionEnd: string;
+    sessionEnd: Date = new Date();
```
[the remark this answers is not in the document]
Comment (RamonG92) [reply src/entities/epn_role_session.entity.ts]: Same solution

File: src/migration/1539014853555-CreateDatabase.ts (hunk 3 above)
Comment (RamonG92) [reply src/migration/1539014853555-CreateDatabase.ts]: Removed the unnecessary migrations, only added the initial create schema migration. And made changes to the README so it tells that schema:log can be used.

**Pull request merged**